bench/simple/Main.hs

```diff
@@ -12,7 +12,8 @@ import Data.Char
 import Data.Ord  (comparing)
 import Data.List (maximumBy)
 
-import Data.Vector.Unboxed.Mutable
+import qualified Data.Vector.Unboxed.Mutable as U
+import Data.Vector.Unboxed.Mutable (MVector, Unbox)
```

Comment:
Please change that `U` qualified name to `UVector` which is better for documentation purposes and less likely to clash with other names. Otherwise looks good!

Reply:
Done.